src/SMAIAXBackend.API/Endpoints/User/UserEndpoints.cs

```diff
@@ -0,0 +1,23 @@
+﻿using SMAIAXBackend.Application.DTOs;
+
+namespace SMAIAXBackend.API.Endpoints.User;
+
+public static class UserEndpoints
+{
+    public static WebApplication MapUserEndpoints(this WebApplication app)
+    {
+        const string contentType = "application/json";
+
+        var group = app.MapGroup("/api/user")
+            .WithTags("User");
```

Comment:
add `.RequireAuthorization();` to protect the api endpoints. Also write an integration test that explicitly checks if an endpoint can only be accessed with an Access Token.